content/en/tracing/trace_queries/_index.md

```diff
@@ -0,0 +1,111 @@
+
+A trace query is composed of two or more [span queries](#span-queries), joined by [trace query operators](#trace-query-operators).
+
+### Span queries
+
+Query for spans from a specific environment, service, or endpoint using the [Span query syntax][1]. Use autocomplete suggestions to view facets and recent queries.
+
+Click **Add another span query** to add a span query and use it in the trace query statement.
+
+### Trace query operators
+
+Combine multiple span queries, labeled `a`, `b`, `c`, and so on, into a trace query in the **Traces where spans** field, using operators between the letters that represent each span query:
+
+{{< img src="/tracing/trace_queries/joined_span_queries.png" alt="Span queries combined into a trace query" style="width:70%;" >}}
+
+| Operator | Description | Example |
+|-----|-----|-----|
+| `&&` | **And**: Both spans are in the trace | Traces that contain spans from the service `web-store` and spans from the service `payments-go`: <br/>`service:web-store && service:payments-go` |
+| `\|\|` | **Or**: One or the other span are in the trace | Traces that contain spans from the service `web-store` or from the service `mobile-store`: <br/>`service:web-store && service:mobile-store` |
+| `->` | **Indirect relationship**: Traces that contain a span matching the left query that is upstream of spans matching the right query | Traces where the service `checkoutservice` is upstream of the service `quoteservice`: <br/>`service:checkoutservice -> service:quoteservice` |
+| `=>` | **Direct relationship**: Traces that contain a span matching the left query that is the direct parent of a span matching the right query | Traces where the service `checkoutservice` is directly calling the service `shippingservice`: <br/>`service:checkoutservice => service:shippingservice` |
+
+## Flow Map
+
+{{< img src="tracing/trace_queries/trace_flow_map.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Trace Flow Map" >}}
+
+The Flow Map helps you understand the request path and service dependencies from the resulting traces that match the Trace Query. Use the map to identify error paths, unusual service dependencies, or abnormally high request rates to a database.
+
+**Note**: The Flow Map is powered by [a sample of the ingested traffic](#the-data-that-trace-queries-are-based-on).
+
+Service nodes that match span queries are highlighted to show you which parts of the trace your query conditions are targeting.
+
+To get more information about **a single service**, hover on the service's node to see its metrics for request rate and error rate. To see metrics for the request rate and the error rate **between two services**, hover on an edge connecting the two services.
+
+Click with the map nodes to filter our traces that do not contain a dependency on the targeted service.
```

Comment:
I couldn't quite figure out what this was saying. Is it something like "To filter out traces that do not contain a dependency (or "aren't involved" on the targeted service (do something)"?

Reply:
```suggestion
To filter out traces that do not contain a dependency on a particular service, click on the service's node on the map.
```